app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
```

Comment:
The handling of `NotWearingMaskError` should ensure consistency with the error message defined in `app/cafe.py`. Since the error message there includes a placeholder `{masks_to_buy}`, ensure that the logic here aligns with that message format. This might involve adjusting the error message or the logic to dynamically include the number of masks needed.